docs/whatsnew/5.1.rst

```diff
@@ -95,6 +96,21 @@ effectively allowing tables within tables. This applies to |Column|,
 units. These structured data columns can be stored in ECSV, FITS, and HDF5
 formats.
 
+.. _whatsnew-fitters:
+
+New fitters have been added
+===========================
```

Comment:
Since this document covers the entire package, maybe should clarify the fitter is for models?
```suggestion
New model fitters have been added
=================================
```